en/VehiclePosition.md

```diff
@@ -0,0 +1,38 @@
+### VehiclePosition
+
+Following are the minimal suggested field requirements for a VehiclePostions feed to provide consumers with high-quality data (e.g., for generating predictions)
+
+| Field name | Required? | Description |

```

Comment:
The sentence above says that the following are suggestions, but the table has a "Required?" column. If they are required then it should be in the spec, if not then the column should be renamed to "Suggested".

Reply:
Fixed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/c6155c04893be21b63ad3d41af3cd057178c478a.